addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/TechsJsonParser.java

```diff
@@ -104,10 +105,12 @@ TechData parse(String categories, List<String> technologies, boolean createIcons
         futures.forEach(CompletableFuture::join);
         executor.shutdown();
         Instant finish = Instant.now();
+        Long loadTime = Duration.between(start, finish).toMillis();
```

Comment:
Just noticed

Shall I inline this instead of the intermediate store?

```suggestion
        Long loadTime = Duration.between(start, Instant.now()).toMillis();
```